cypress.dev.json

```diff
@@ -0,0 +1,7 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "env": {
```

Comment:
These environment variables are available in the test environment when Travis is running. For local development, we add them explicitly here because we can not `source .env` our configuration in npm scripts. It has no effect.